scripts/scripts.js

```diff
@@ -0,0 +1,227 @@
+  test.clickedLog(event); // log what is clicked
+
+  dom.macMenu.classList.toggle("hidden-wrapper--show");
+});
+
+///////////////
+/// About This Mac
+dom.menuAbout.addEventListener("click", (event) => {
+  test.clickedLog(event); // log what is clicked
+
+  dom.aboutWindow.classList.add("hidden-wrapper--show");
+  dom.macMenu.classList.remove("hidden-wrapper--show");
+});
+
+dom.aboutCloseBtn.addEventListener("click", (event) => {
+  test.clickedLog(event); // log what is clicked
+
+  dom.aboutWindow.classList.remove("hidden-wrapper--show");
+});
+
+/////////////////////////////////////////////////////////
+/// Apps
+///
+const hideAllWindows = () => {
+  for (let i = 0; i < dom.windows.length; i++) {
+    const window = dom.windows[i];
+    if (window.classList.contains("hidden-wrapper--show")) {
+      window.classList.remove("hidden-wrapper--show");
+    }
+  }
+  dom.playerVideo.pause(); // pause bunny video when window hidden
+};
+
+///////////////////////
+/// Notes app
+///
+
+const savedNotesArr = [];
+
+///////// Read Me note
+nf.saveNewNote(
+  "Read Me",
+  `Hello world, this is a dummy note from Oscar! \n
+  Welcome to my Macintosh web replica! \n
+  You can create new note and retrieve saved note content. \n
+  Go on! Create as many notes as you want! \n
+  \n
+  \n
+  Delete is out of scope... :)`,
+  savedNotesArr
+);
+
+///////
+// open & close
+dom.notesIcon.addEventListener("dblclick", (event) => {
+  test.clickedLog(event); // log what is clicked
+  hideAllWindows();
+  dom.notesWindow.classList.add("hidden-wrapper--show");
+
+  nf.displaySavedNotes(savedNotesArr);
+  nf.saveNotesClickListen(savedNotesArr);
+});
+dom.notesCloseBtn.addEventListener("click", (event) => {
+  test.clickedLog(event); // log what is clicked
+  dom.notesWindow.classList.remove("hidden-wrapper--show");
+
+  dom.notesForm.reset();
+});
+
+///////////////
+
+// Save btn clicked
+dom.notesSave.addEventListener("click", (event) => {
+  test.clickedLog(event); // log what is clicked
+
+  // empty entries -> report & exit
+  if (!dom.notesForm.checkValidity()) {
+    dom.notesForm.reportValidity();
+    return;
+  }
+
+  // prevent page from reload
+  event.preventDefault();
+
+  const notesFormData = new FormData(dom.notesForm, dom.notesSave); // get new data every click
+  const noteTitle = notesFormData.get("noteTitle"); // name of <input>
+  const noteContent = notesFormData.get("noteContent"); // name of <input>
+
+  console.log("noteTitle saved: ", noteTitle);
+  console.log("noteContent saved: ", noteContent);
+
+  nf.saveNewNote(noteTitle, noteContent, savedNotesArr);
+
+  nf.displaySavedNotes(savedNotesArr);
+  nf.saveNotesClickListen(savedNotesArr);
+
+  dom.notesForm.reset(); // clear inputs
+});
+
+///////////////////////
+/// Player app
+///
+dom.playerIcon.addEventListener("dblclick", (event) => {
+  test.clickedLog(event); // log what is clicked
+  hideAllWindows();
+  dom.playerWindow.classList.add("hidden-wrapper--show");
```

Comment:
if all icons need the same functionality, you can select them all using .querySelecotrAll, and then do app.forEach((app) => {
   app.classList.remove("classname)
})